rpm distribution generates artifact errors

See http://build-us-00.elastic.co/job/es_core_master_regression/2919/console

[ERROR] Failed to execute goal org.codehaus.mojo:rpm-maven-plugin:2.1.3:attached-rpm (attach-rpm) on project elasticsearch-rpm: Execution attach-rpm of goal org.codehaus.mojo:rpm-maven-plugin:2.1.3:attached-rpm failed: For artifact {org.elasticsearch.distribution:elasticsearch-rpm:2.0.0-beta1-SNAPSHOT:rpm}: An attached artifact must have a different ID than its corresponding main artifact. -> 

Looks like its angry that its elasticsearch and not elasticsearch-rpm?

--- a/distribution/rpm/pom.xml
+++ b/distribution/rpm/pom.xml
@@ -339,31 +339,6 @@
 
     <profiles>
         <profile>
-            <id>release</id>
-            <activation>
-                <property>
-                    <name>package.rpm</name>
-                    <value>true</value>
-                </property>
-            </activation>
-            <build>
-                <plugins>
-                    <plugin>
-                        <groupId>org.codehaus.mojo</groupId>
-                        <artifactId>rpm-maven-plugin</artifactId>
-                        <executions>
-                            <execution>
-                                <id>attach-rpm</id>
-                                <goals>
-                                    <goal>attached-rpm</goal>
-                                </goals>
-                            </execution>
-                        </executions>
-                    </plugin>
-                </plugins>
-            </build>
-        </profile>
-        <profile>
             <id>sign-rpm</id>
             <activation>
                 <property>

